packages/docs/src/page-configs/ui-elements/select/examples/Slots.vue

```diff
@@ -17,6 +18,7 @@
       label="Prepend inner slot"
       :options="options"
       v-model="value"
+      bordered
```

Comment:
```suggestion
```

Any reasons to make first two selects bordered?

Reply:
removed this option